README.md

```diff
@@ -24,6 +24,11 @@ e.g.
 docker push quay.io/kiegroup/kie-cloud-operator:<version>
 ```
 
+### Note
+If your images pushed to quay.io may are set as private you need to that user needs to have the quay.io pull secret set in OCP 
```

Comment:
you need to be aware of user needs to have.. ?

Reply:
A suggestion from @jakubschwan above

Reply:
needs rephrase, text is a little bit confusing.
maybe something line: 

`If the quay.io repository where the images were pushed is private, a pull secret will need to be configured otherwise...